docs/features/metrics.md

```diff
@@ -68,7 +68,7 @@ Once instructors have set up risk metrics for their course, students that are id
 
 Every row in the watchlist represents a particular instance of a student who meets one or more of the risk conditions. A single student can appear in multiple watchlist instances if they are identified for new risk conditions on separate occassions of loading the watchlist. 
 
-For example, let's look at Jane Doe in the image above. Upon loading the watchlist, Jane appears in a watchlist instance for using `3 grace days` before the instructor-specified date and for having `2 low scores` below the instructor-specified threshold. If Jane later receives another score below the threshold, a new instance will appear for Jane when the instructor reloads the watchlist. Jane now appears twice in the watchlist, once in an instance with `3 grace days` and `2 low scores`, and once in an instance with `3 grace days` and `3 low scores`.
+For example, let's look at Jane Doe in the image above. Upon loading the watchlist, Jane appears in a watchlist instance for using `3 grace days` before the instructor-specified date and for having `2 low score` below the instructor-specified threshold. If Jane later receives another score below the threshold, a new instance will appear for Jane when the instructor reloads the watchlist. Jane now appears twice in the watchlist, once in an instance with `3 grace days` and `2 low score`, and once in an instance with `3 grace days` and `3 low score`.
```

Comment:
Not a big deal, but should we just change the wording in the actual feature to say "low scores" instead of "low score"? Just for consistency since "grace days" is pluralized. Let me know what you think!

Reply:
That's a good point. Currently we have the following:
![image](https://user-images.githubusercontent.com/31053044/115465836-3309a500-a1fd-11eb-9bb2-fc861e30c47f.png)

I'm thinking perhaps we should have a seperate PR to address the plural / singular tags? 
@victorhuangwq what do you think?

Reply:
yeah we will add it to the backlog. We won't address it in this PR

Reply:
Cool, should we just keep `low scores` as plural in the docs then?

Reply:
Let's make it low scores in the docs. We should just change the UI to follow that as well afterwards